Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -53,6 +53,7 @@ if [ ! -n "${NIGHTLY}" ] && [ ! -n "${BUCKET_UPLOAD}" ]; then
     fi
   fi
 else
+  IS_FORCE_UPLOAD=false
```

Comment:
?

Reply:
she should merge from master and it'll be fixed